deps.bzl

```diff
@@ -1,479 +0,0 @@
-load("@bazel_skylib//lib:modules.bzl", "modules")
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
-
-# The podman version used in tests and distributed with the buildbuddy executor image.
-# When changing this version, a new release of podman-static may be needed.
-# See dockerfiles/executor_image/README.md for instructions.
-# The checksums below will also need to be updated.
-PODMAN_VERSION = "v5.5.0"
```

Comment:
If we need to read this from BUILD files we would have to keep it in a .bzl file